lib/lte_link_control/Kconfig

```diff
@@ -54,28 +54,35 @@ config LTE_UNLOCK_PLMN
 	help
 		Disable PLMN locks for network selection.
 
+menuconfig LTE_PSM_REQ
+	bool "Enable requesting use of PSM."
```

Comment:
I know this is matching the eDRX text, but I think it can be improved a bit. The punctuation should be removed, and it can be reworded to something like `Enable PSM requests`. If you want, the same can be done for eDRX.